src/server/routes/email.ts

```diff
@@ -0,0 +1,25 @@
+import { Router, Request, Response } from 'express';
+import { sendExampleEmail } from '@/email';
+
+const router = Router();
+
+router.get('/send-example-email?:to', async (req: Request, res: Response) => {
```

Comment:
Is this endpoint accessible to the Internet?

We should add protection around this to avoid abuse.

Reply:
It's not public until the Fastly config also exists. But I discussed this with Mahesh and he pointed out that even without that config there's a risk it could be added by mistake later on.

I'm going to remove this code for now. It was useful as an example for how an email could be sent but that now this also exists in the README so this code is redundant

Reply:
👍  for removal

> It's not public until the Fastly config also exists

The load balancer address might be accessible to the internet depending on the security group config.